.github/actions/slack_codescan_notification/action.yml

```diff
@@ -4,6 +4,9 @@ inputs:
   title:
     description: "Title of the failed workflow"
     required: true
+  warningOnly:
```

Comment:
this action now handles both warning and error scenarios so don't need different configuration for them